examples/map/difference_images.py

```diff
@@ -0,0 +1,64 @@
+"""
+================
```

Comment:
These lines need to be the same length as the title.

Reply:
Done (I think)